packages/cozy-client/src/links/StackLink.spec.js

```diff
@@ -36,31 +39,43 @@ describe('StackLink', () => {
       stackClient.collection().find.mockReset()
       await link.request(query)
       expect(stackClient.collection().find).toHaveBeenCalled()
-      expect(stackClient.collection).toHaveBeenCalledWith('io.cozy.todos')
+      expect(stackClient.collection).toHaveBeenCalledWith(
+        'io.cozy.todos',
+        expect.anything()
+      )
```

Comment:
Since this commit actually fix tests from previous commit,  it should be rebased within the commit where it breaks